### PR TITLE
fix(store_server_info): storing invites and bans work properly

### PR DESCRIPTION
**Pull request by DanTheComputerMan**
Added delay to allow bans and invites to fetch. Previously returned undefined or 0.

fixes: https://github.com/dbm-network/mods/issues/231

### CODE REVIEW

**Comment by DanTheComputerMan** [comment]
Updated to async/await.

File: actions/store_server_info.js [hunk 1]
```diff
@@ -465,33 +474,29 @@ module.exports = {
...
+			default: // Fixed Spacing. Fixed by CoolGuy in 1.9.7
+				break;
 		};
 		if (result !== undefined) {
-			const storage = parseInt(data.storage);
-			const varName2 = this.evalMessage(data.varName2, cache);
-			this.storeValue(result, storage, varName2, cache);
+			const storage3 = parseInt(data.storage);
+			const varName4 = this.evalMessage(data.varName2, cache);
+			this.storeValue(result, storage3, varName4, cache);
```
Comment (Almeeida) [inline actions/store_server_info.js]: These changes aren't required.

File: actions/store_server_info.js [hunk 2]
```diff
@@ -465,33 +474,29 @@ module.exports = {
...
-					result = targetServer.explicitContentFilter;
-					break;
-			     default:
-				     break;
+			case 41: //	Collection of banned users. Fixed by CoolGuy in 1.9.7
+				const bans = await targetServer.fetchBans();
+				result = bans.array();
+				const storage = parseInt(data.storage);
+				const varName2 = this.evalMessage(data.varName2, cache);
+				this.storeValue(result, storage, varName2, cache);
```
Comment (Almeeida) [inline actions/store_server_info.js]: Not needed here

File: actions/store_server_info.js [hunk 3]
```diff
@@ -465,33 +474,29 @@ module.exports = {
...
+				break;
+			case 42: //	Collection of guild invites. Fixed by CoolGuy in 1.9.7
+				const invites = await targetServer.fetchInvites();
+				result = invites.array();
+				const storage2 = parseInt(data.storage);
+				const varName3 = this.evalMessage(data.varName2, cache);
+				this.storeValue(result, storage2, varName3, cache);
```
Comment (Almeeida) [inline actions/store_server_info.js]: Not needed here.
It's also missing a `break` statement.

**Comment by DanTheComputerMan** [comment]
Updated.

File: actions/store_server_info.js [hunk 4]
```diff
@@ -315,178 +324,175 @@ module.exports = {
...
+				const storage = parseInt(data.storage);
+				const varName2 = this.evalMessage(data.varName2, cache);
+				this.storeValue(result, storage, varName2, cache);
+				break; }
+			case 42: { // Collection of guild invites. Fixed by CoolGuy in 1.9.7
+				const invites = await targetServer.fetchInvites();
+				result = invites.array();
+				const storage = parseInt(data.storage);
+				const varName2 = this.evalMessage(data.varName2, cache);
+				this.storeValue(result, storage, varName2, cache);
```
Comment (Almeeida) [inline actions/store_server_info.js]: Not needed here

File: actions/store_server_info.js [hunk 5]
```diff
@@ -315,178 +324,175 @@ module.exports = {
...
-					break;
-			     default:
-				     break;
+				break; }
+			case 41: { // Collection of banned users. Fixed by CoolGuy in 1.9.7
+				const bans = await targetServer.fetchBans();
+				result = bans.array();
+				const storage = parseInt(data.storage);
+				const varName2 = this.evalMessage(data.varName2, cache);
+				this.storeValue(result, storage, varName2, cache);
```
Comment (Almeeida) [inline actions/store_server_info.js]: Not needed here

**Comment by DanTheComputerMan** [comment]
Removed unneeded variable assignments and await all .fetchMembers() methods.

**Pull request merged**